docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -34,6 +36,15 @@ Reasons to prefer docker:
 * uses less resources (RAM, CPU and disk)
 * improved performance
 * host may freely access processes within the guest (e.g. for debugging)
+* no extra firewall ports to open on your development platform
+
+Prerequisites for Vagrnt
+-------------------------
+
+#. Install vagrant and ansible. Vagrant will manage your development vms or conatiners and ansible
+   is used to configure the system once it is running.
+
+      $ sudo dnf install vagrant ansible
```

Comment:
When built this won't be escaped properly. To do that add a `::` to the end of L#45. You can proof these changes by [building the docs locally](http://docs.pulpproject.org/dev-guide/contributing/documenting.html#build-docs-locally).